main.py

```diff
@@ -1,10 +1,69 @@
 import init_django_orm  # noqa: F401
 
+import json
```

Comment:
Change the order of your imports according to `pep8`. Here `json` should be first.